web/themes/joinup/joinup_theme.theme

```diff
@@ -1,6 +1,42 @@
 <?php
-
 /**
  * @file
  * Functions to support theming in the Joinup theme.
  */
+
+use Drupal\image\Entity\ImageStyle;
+use \Drupal\file\Entity\File;
+
+/**
+ * Implements hook_preprocess_page().
+ */
+function joinup_theme_preprocess_page(&$variables) {
+  // Set page title.
```

Comment:
Yes, by default the title is only accessible from within a block...
As depending on the page the layout will change considerably (blue header entity edit pages vs. collection page) I'd prefer to have a simple variable that contains the page title to print.